test/offset_test.rb

```diff
@@ -0,0 +1,31 @@
+require 'minitest/autorun'
+require 'minitest/pride'
+require 'mocha/minitest'
+require_relative '../lib/offset'
+
+class OffsetTest < Minitest::Test 
+
+  def test_it_exists 
+    offset = Offset.new('02715')
+    assert_instance_of Offset, offset
+  end 
+
+  def test_it_has_attributes
+    offset1 = Offset.new('040895')
+    assert_equal '040895', offset1.date
+
+    offset2 = mock('offset')
+    offset2.stubs(:date).returns('100120')
```

Comment:
mocks an `offset` object and stubs out the `.date` reader method for cases when a date is not passed as an argument